src/js/selfVoicing.js

```diff
@@ -65,10 +65,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 "method": "click",
                 "args": ["{that}.toggleState"]
             },
-            // TODO: The following listeners can be removed after switching to use model relay
-            "onCreate.setLabel": "{that}.setLabel",
+            // Need to call the handlers onCreate and exclude "init" on the modelListeners
+            // because the underlying tooltip widget isn't finished at initialization
             "onCreate.setTooltip": "{that}.setTooltip",
-            "onCreate.setMuteStyle": "{that}.setMuteStyle",
             "onCreate.clearQueue": {
                 listener: "gpii.firstDiscovery.selfVoicing.clearQueue",
                 args: ["{that}"]
```

Comment:
Can we defined an invoker for clearQueue to be shared for avoiding the repetition of passing the same set of arguments in to the same func? Thanks.